<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>690</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>굴림체</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>BurpSuite Certification Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="logEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>441</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>13</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP addr:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>13</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8080</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>12</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Device List:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QListWidget" name="deviceList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
